packages/database/src/db/connection.ts

```diff
@@ -0,0 +1,14 @@
+import {Mode} from 'drizzle-orm/mysql-core';
+import * as schema from './schema';
+import {drizzle} from 'drizzle-orm/mysql2';
+import mysql from 'mysql2/promise';
+import {env} from 'node:process';
+
+const poolConnection = mysql.createPool({
+  uri: env['DATABASE_URL'],
+});
+
+export default drizzle(poolConnection, {
```

Comment:
To be sure (in keeping with last week's PR): would it be possible to not use a `default export` but just an `export`?